src/app/campaign/campaign.component.spec.ts

```diff
@@ -1,22 +1,123 @@
-import { CampaignComponent } from './campaign.component';
-import { CampaignStoreService, CampaignState, FlightState, AccountService, AdvertiserService } from '../core';
+import { TestBed, async, ComponentFixture } from '@angular/core/testing';
```

Comment:
This component spec didn't have a proper TestBed, so I worked one up and set up all the dependencies.